Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -436,7 +439,7 @@ foreach ($process in $ServerProcess) {
 
         # Clear out modules from the allow list
         foreach ($module in $ModuleAllowList) {
-            $ProcessModules = $ProcessModules | Where-Object { $_.ModuleName -ne $module }
+            $ProcessModules = $ProcessModules | Where-Object { $_.ModuleName -ne $module -and $_.ModuleName -ne $($module.Replace(".dll", ".ni.dll")) }
```

Comment:
you currently have these that will not convert.

```
$ModuleAllowList.add("SCCOLE2.DLL")
$ModuleAllowList.add("SCCSD.DLL")
$ModuleAllowList.add("SCCXT.DLL")
```

Reply:
I think it is not a big issue, because those DLLs are not .net libraries so it is not need it to check ni files, but I will change all DLL to dll and we will have all files in the same format.